packages/core/L3/nodejs/tools/JestSupport.ts

```diff
@@ -651,12 +651,15 @@ export class JestSupport extends Construct {
           type: PackageDependencyType.DEV,
         });
 
+        // https://kulshekhar.github.io/ts-jest/docs/getting-started/options
         const additionalConfig = {
-          preset: "ts-jest",
-          globals: {
-            "ts-jest": {
-              tsconfig: "tsconfig.json",
-            },
+          transform: {
+            '^.+\\.tsx?$': [
+              'ts-jest',
+              {
+                tsconfig: "tsconfig.json",
+              },
+            ],
```

Comment:
@Pagebakers This is a perfect example of a fix that would propagate to downstream consumers of this package, and everyone would get the config fixed automatically.

Reply:
Nice!